src/test/javascript/spec/component/exam/manage/exam-exercise-import.component.spec.ts

```diff
@@ -299,4 +299,123 @@ describe('Exam Exercise Import Component', () => {
         expect(component.getExerciseIcon(quizExercise)).toEqual(faCheckDouble);
         expect(component.getExerciseIcon(fileUploadExercise)).toEqual(faFileUpload);
     });
+
+    describe(`Programming exercise import validation`, () => {
```

Comment:
```suggestion
    describe('Programming exercise import validation', () => {
```
I would not use template strings where normal ones suffice